integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/synapseconfig/vfsTransport/ESBJAVA4770/axis2.xml

```diff
@@ -867,8 +867,8 @@ check com.sun.mail.pop3 and com.sun.mail.imap package documentation for more det
 
             <handler name="HTTPLocationBasedDispatcher"
                      class="org.apache.axis2.dispatchers.HTTPLocationBasedDispatcher"/>
-            <handler name="MultitenantDispatcher"
-                     class="org.wso2.carbon.tenant.dispatcher.MultitenantDispatcher"/>
+<!--            <handler name="MultitenantDispatcher"-->
+<!--                     class="org.wso2.carbon.tenant.dispatcher.MultitenantDispatcher"/>-->
```

Comment:
Shall we remove these if not required?

Reply:
will do in the next PR

Reply:
Fixed as part of https://github.com/wso2/micro-integrator/pull/516